.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -30,6 +29,7 @@ steps:
     steps:
       - label: ":ubuntu: Auditbeat Unit Tests"
         command: |
+          set -euo pipefail
```

Comment:
While it doesn't hurt to have it, `set -euo pipefail` is not needed for the simple two line script here, because it will trigger a hard exit only if `cd auditbeat` fails (why never will).

The second command (`mage build unitTest`), since it's the last command, will cause a build failure as expected if it returns an error, even without `set -euo pipefail`.